runtime/fundamentals/security.md

```diff
@@ -256,8 +257,11 @@ deno run -E script.ts
 # or
 deno run --allow-env script.ts
 
-# Allow HOME and FOO environment variable
+# Allow HOME and FOO environment variables
 deno run --allow-env=HOME,FOO script.ts
+
+# Allow access to all environment variables starting with AWS_
+deno run --allow-env=AWS_* script.ts
```

Comment:
Probably it's better to surround the value with quotes as some shells try to expand `*` and fail:

```suggestion
deno run --allow-env="AWS_*" script.ts
```

Reply:
The blog post doesn't quote them...